draft-terbu-sd-jwt-vc.md

```diff
@@ -38,46 +38,58 @@ express Verifiable Credentials with JSON payload based on the securing mechanism
 Holders which can present Verifiable Credentials to Verifiers typically in form
-of Verifiable Presentations.
+of Verifiable Presentations which are secure envelops that contain Verifiable
+Credentials addressed to a specific audience.
 
 These relationships are described by the three-party-model which involves the
 following parties:
 
-1. Issuer: The entity that issues the Verifiable Credential to the Holder, who
-is tthe person or entity being issued the credential.
+1. Issuer: The entity that issues the Verifiable Credential to the Holder.
 1. Verifier: The entity that verifies the Verifiable Credential presented by
 the Subject, for example to prove eligibility to access certain services.
 1. Holder: The person or entity being issued the Verifiable Credential, who
  may present the Verifiable Credential to a Verifier for verification.
 
-Verifiers have to trust Issuers to make
+In the three-party-model, Verifiers have to trust Issuers to make
 trustworthy statements about the Subject and they can additionally require that
 the Holder provides a proof that they are the intended Holder of the Verifiable
 Credential which can important for security reasons. This is only possible if
 an Issuer binds the Verifiable Credential to a specific Holder at the time of
-issuance.
+issuance. This process is referred to as Holder Binding and is further
+described in [SD-JWT].
+
+The three-party-model, i.e., actors, Verifiable Credentials and Verifiable
+Presentations, are further described in [VCDM2.0]. However, this specification
+focuses on a specific version of the three-party-model which can have
+different features but will provide a representation of the model
+described in [VCDM2.0].
+
+## Rationale
 
 JSON Web Tokens (JWTs) [@!RFC7519] can in principle be used to express
 Verifiable Credentials in a way that is easy to understand and process as it
 builds upon established web primitives. However, JWTs do not support selective
 disclosure, i.e., the ability to disclose only a subset of the claims contained
-in the JWT, in the three-party-model as described above. This is a common problem in the so-called three-party model: An
-Issuer creates a Verifiable Credential for some End-User (Holder), who then
-presents this credential to multiple Verifiers. A credential might contain a
-large number of claims, but the Holder typically only wants to disclose a subset
-of these claims to a Verifier. In this case, the Holder would have to receive a
-new JWT from the Issuer, containing only the claims that should be
-disclosed, for each interaction with a new Verifier. This is inefficient,
+in the JWT, in the three-party-model as described above. This is a common problem
+in the three-party model: An Issuer creates a Verifiable Credential for
+some End-User (Holder), who then presents this credential to multiple Verifiers.
+A Verifiable Credential might contain a large number of claims, but the Holder
```

Comment:
Do we really need to explain that in lengths?

Reply:
I didn't add this language in this PR. I think @danielfett wanted to fix that in a separate PR.